Unified Bug Dataset added to Data sets list

zizo007 opened the pull request:
New entry in the data sets section, named Unified Bug Dataset.

zizo007 committed:
Unified Bug Dataset added.

dspinellis commented:
Please remove the link to `bugcatcher.stca.herts.ac.uk/bugcatchers/`.  I am getting a `Potential Security Risk Ahead` warning, and maybe Lint also fails because of this.

zizo007 committed:
Bugcatchers link removed

Bugcatchers link removed due to Potential Security Risk Ahead warning.

zizo007 committed:
Ending punctuation added.

zizo007 commented:
Removed the link and added a missing punctuation to the end of the line. Now, I don't know what the 'Awesome list must reside in a valid git repository  remark-lint:awesome/github' error means.

dspinellis commented:
Thank you!
